state/protocol/protocol_state/mutator.go

```diff
@@ -269,3 +273,16 @@ func (m *stateMutator) transitionToEpochFallbackMode(results []*flow.ExecutionRe
 	}
 	return dbUpdates, nil
 }
+
+// epochFallbackTriggeredByIncorporatingCandidate checks whether incorporating the input block
+// would trigger epoch fallback mode (EFM) along the current fork. In particular, we trigger epoch
+// fallback mode when:
+//  1. The next epoch has not been committed as of B (EpochPhase < flow.EpochPhaseCommitted) AND
```

Comment:
would be inclined to use the same representation as in code: https://github.com/onflow/flow-go/blob/59165d936d14e6ed2c8ec4dac3572c1830e2b421/state/protocol/protocol_state/mutator.go#L284

```suggestion
//  1. The next epoch has not been committed as of B (EpochPhase ≠ flow.EpochPhaseCommitted) AND
```
This holds independently of the internal order of `EpochPhase`